README.md

```diff
@@ -1,12 +1,12 @@
-# [Stanford Starter](https://github.com/SU-SWS/stanford_starter)
+# [vpue_undergrad_subtheme](https://github.com/SU-SWS/vpue_undergrad_subtheme)
```

Comment:
This is, I think, the pretty name we will see in the UI. Let's make this just say "VPUE Undergrad"

Reply:
Agreed - that looks much nicer in the UI